sdk/scala-sdk-protobuf-testkit/src/main/scala/kalix/scalasdk/testkit/impl/TestKitActionContext.scala

```diff
@@ -33,5 +34,5 @@ final class TestKitActionContext(
 
   override def eventSubject = metadata.get("ce-subject")
   override def getGrpcClient[T](clientClass: Class[T], service: String): T = getComponentGrpcClient(clientClass)
-
+  override def getOpenTelemetryTracer: Option[Tracer] = None
```

Comment:
So, this API doesn't exist yet in the Scala SDK? 
Maybe better to go for the Noops one we discussed the other day and not return Option.

Reply:
we have option in the java-proto. How about we use the same here and then we change to NoOps in both? or you think is better we change this now?

Reply:
Better to not add an API that we plan to deprecate. 

But we need to decide if the NoOps is what we want. At first, it sounds like a good idea. But if we take that path, we will need to deprecate the method in the java sdk. Not a big issue, but better to do it now than later. 

If we decide to deprecate it, we need to come up with a new method name and then better to also use that new name here.

Reply:
Ok. I'll go for the NoOps now. If you have any objections please let me know.